<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BolusCalculator</class>
 <widget class="QWidget" name="BolusCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelGlucose">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Glucose Level (mmol/L):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputGlucose">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelCarbs">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Carbohydrates (g):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputCarbs">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCalculate">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>80</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate Bolus</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelResult">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Recommended Dose (u):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="outputResult">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="overrideCheckbox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>160</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Override Dose</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="overrideDoseInput">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>220</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Enter dose</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOverrideConfirm">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>220</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm Override</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDeliver">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>270</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Deliver</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logoButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#logoButton {
    background-color: #005F87;
    color: black;
    font-weight: bold;
    font-size: 16px;
    padding: 6px 12px;
    border-radius: 6px;
    border: 1px solid #007ACC;
}

QPushButton#logoButton:hover {
    background-color: #1E90FF;
}

QPushButton#logoButton:pressed {
    background-color: #007ACC;
    color: white;
}
</string>
   </property>
   <property name="text">
    <string>T</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>9</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Bolus calculator</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelBolus">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>270</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel Bolus</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
